iwyu.cc

```diff
@@ -87,58 +87,63 @@
-#include <set>                          // for set, set<>::iterator, swap
-#include <string>                       // for string, operator+, etc
-#include <utility>                      // for pair
-#include <vector>                       // for vector, swap
-
-#include "clang/AST/ASTConsumer.h"
-#include "clang/AST/ASTContext.h"
-#include "clang/AST/Decl.h"
-#include "clang/AST/DeclBase.h"
-#include "clang/AST/DeclCXX.h"
-#include "clang/AST/DeclTemplate.h"
-#include "clang/AST/ExprConcepts.h"
-#include "clang/AST/NestedNameSpecifier.h"
-#include "clang/AST/OperationKinds.h"
-#include "clang/AST/RecursiveASTVisitor.h"
-#include "clang/AST/TemplateBase.h"
-#include "clang/AST/Type.h"
-#include "clang/AST/TypeLoc.h"
-#include "clang/Basic/FileEntry.h"
-#include "clang/Basic/SourceLocation.h"
-#include "clang/Basic/TypeTraits.h"
-#include "clang/Frontend/CompilerInstance.h"
-#include "clang/Frontend/FrontendAction.h"
-#include "clang/Lex/Preprocessor.h"
-#include "clang/Lex/PreprocessorOptions.h"
-#include "clang/Sema/Sema.h"
-#include "iwyu_ast_util.h"
-#include "iwyu_cache.h"
-#include "iwyu_driver.h"
-#include "iwyu_globals.h"
-#include "iwyu_lexer_utils.h"
-#include "iwyu_location_util.h"
-#include "iwyu_output.h"
-#include "iwyu_path_util.h"
-#include "iwyu_port.h"  // for CHECK_
-#include "iwyu_preprocessor.h"
-#include "iwyu_stl_util.h"
-#include "iwyu_string_util.h"
-#include "iwyu_use_flags.h"
-#include "iwyu_verrs.h"
-#include "llvm/Support/Casting.h"
-#include "llvm/Support/TargetSelect.h"
-#include "llvm/Support/raw_ostream.h"
+#include <stdio.h>                              // for snprintf
+#include <cstdlib>                              // for exit, size_t, EXIT_FA...
+#include <functional>                           // for function
+#include <map>                                  // for map, operator!=, _Rb_...
+#include <memory>                               // for unique_ptr, allocator
+#include <set>                                  // for set
+#include <string>                               // for basic_string, operator+
+#include <utility>                              // for pair, swap
+#include <vector>                               // for vector
+
+#include "clang/AST/ASTConsumer.h"              // for ASTConsumer
+#include "clang/AST/ASTContext.h"               // for ASTContext
+#include "clang/AST/Attr.h"                     // for Attr
+#include "clang/AST/Decl.h"                     // for FunctionDecl, NamedDecl
+#include "clang/AST/DeclBase.h"                 // for operator!=, Decl, Dec...
+#include "clang/AST/DeclCXX.h"                  // for CXXRecordDecl, CXXMet...
+#include "clang/AST/DeclTemplate.h"             // for ClassTemplateSpeciali...
+#include "clang/AST/Expr.h"                     // for CallExpr, Expr, DeclR...
+#include "clang/AST/ExprCXX.h"                  // for CXXConstructExpr, Ove...
+#include "clang/AST/ExprConcepts.h"             // for ConceptSpecialization...
```

Comment:
Shrunk names in the comments doesn't look good.

Reply:
I think we came to the conclusion way back that it was better than `, etc`, which is just five dead characters.